boards/common/atmega/Makefile.dep

```diff
@@ -0,0 +1,2 @@
+# require Makefile.dep for correct dependency management of atmega PCINTs
+include $(RIOTCPU)/$(CPU)/Makefile.dep
```

Comment:
This line is the problem. ```${BOARD}``` and ```${CPU}``` are not known when executing the ```info-boards-supported``` target, which is used to find out which boards an application supports.

Reply:
See this:

```
[kaspar@ng riot.tmp ((ff3e541c09...))]$ make -C tests/xtimer_usleep info-boards-supported
make: Entering directory '/home/kaspar/src/riot.tmp/tests/xtimer_usleep'
/home/kaspar/src/riot.tmp/boards/common/atmega/Makefile.dep:2: /home/kaspar/src/riot.tmp/cpu//Makefile.dep: No such file or directory
make: *** No rule to make target '/home/kaspar/src/riot.tmp/cpu//Makefile.dep'.  Stop.
make: Leaving directory '/home/kaspar/src/riot.tmp/tests/xtimer_usleep'
[kaspar@ng riot.tmp ((ff3e541c09...))]$
```